Makefile

```diff
@@ -50,6 +50,8 @@ else
 endif
 
 local-test: ## Alias for 'make test WORKER_TYPE=qemu'
+	ssh-keygen -q -t rsa -N '' -f ./core/id <<<y >/dev/null 2>&1
```

Comment:
Remind me why we need to create these keys in the workspace and not in the container or in a volume?

Reply:
Why not just enable password authentication on the worker and skip the messy bits with the keys?

Reply:
I couldn't figure out how to automate the password entry every time

Reply:
originally it was so I could get a matching set of keys in both containers before they started up. Generating them inside one container and then getting it into the other container ended up being more messy in the end and needed more logic

Reply:
@klutchell I found a way to make this step unnecessary now.